libopenage/renderer/font_manager.h

```diff
@@ -0,0 +1,86 @@
+// Copyright 2015-2015 the openage authors. See copying.md for legal info.
+
+#ifndef OPENAGE_RENDERER_FONT_MANAGER_H_
+#define OPENAGE_RENDERER_FONT_MANAGER_H_
+
+#include <ft2build.h>
+#include FT_FREETYPE_H
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+#include "../error/error.h"
+
+namespace openage {
+namespace renderer {
+
+struct freetype_library {
+	FT_Library ft_library;
+
+	freetype_library() {
+		if (FT_Init_FreeType(&this->ft_library) != 0) {
+			throw Error{MSG(err) << "Failed to initialize freetype library."};
+		}
+	}
+
+	~freetype_library() {
+		FT_Done_FreeType(this->ft_library);
```

Comment:
You probably want to explicitly delete the move and copy constructors/assignments for this struct. Also, this is more likely to be a class than a struct as it does way more then just packing data.